### ISSUE TITLE
3.5.0 installation fails on ubuntu 14.04

### ISSUE THREAD

**Issue by tyler274** [yyuu/pyenv#483]
libssl-dev is installed, as are all the other dependencies listed on the wiki page. 

    pyenv install 3.5.0
    Downloading Python-3.5.0.tgz...
    1->        https://yyuu.github.io/pythons/584e3d5a02692ca52fce505e68ecd77248a6f2c99adf9db144a39087336b0fe0
    Installing Python-3.5.0...
    ERROR: The Python ssl extension was not compiled. Missing the OpenSSL lib?
    
    Please consult to the Wiki page to fix the problem.
    https://github.com/yyuu/pyenv/wiki/Common-build-problems
    
    
    BUILD FAILED (Ubuntu 14.04 using python-build 20151105-4-g9ca7de5)
    
    Inspect or clean up the working tree at /tmp/user/1007/python-build.20151118035852.20127
    Results logged to /tmp/user/1007/python-build.20151118035852.20127.log
    
    Last 10 log lines:
    (cd /home/dabney/.pyenv/versions/3.5.0/share/man/man1; ln -s python3.5.1 python3.1)
    if test "xupgrade" != "xno"  ; then \
                    case upgrade in \
                            upgrade) ensurepip="--upgrade" ;; \
                            install|*) ensurepip="" ;; \
                     esac; \
                     ./python -E -m ensurepip \
                            $ensurepip --root=/ ; \
            fi
    Ignoring ensurepip failure: pip 7.1.2 requires SSL/TLS

**Comment by yyuu** [comment]
> ERROR: The Python ssl extension was not compiled. Missing the OpenSSL lib?
> 
> Please consult to the Wiki page to fix the problem.
> https://github.com/yyuu/pyenv/wiki/Common-build-problems

**Comment by tyler274** [comment]
I just specified that's not helping.

**Comment by yyuu** [comment]
Apparently your system doesn't have OpenSSL header. You must ensure you have `libssl-dev` installed on your system.

https://github.com/yyuu/pyenv/wiki/Common-build-problems#requirements

**Comment by tyler274** [comment]
tyler274@server1:~/work$ sudo apt-get install libssl-dev
Reading package lists... Done
Building dependency tree
Reading state information... Done
libssl-dev is already the newest version.
0 to upgrade, 0 to newly install, 0 to remove and 4 not to upgrade.

**Comment by tyler274** [comment]
hmm, forcing a reinstall via
    sudo apt-get remove libssl-dev
    sudo apt-get install libssl-dev

seems to have fixed it.

**Comment by yyuu** [comment]
:+1: Probably someone broke the package contents?

**Comment by VShangxiao** [comment]
@yyuu 
I got these prints:

error: failed to download Python-3.4.1.tar.xz
BUILD FAILED (Ubuntu 16.04 using python-build 1.0.2-9-g80e769e)

please give a way to fix this~

**Comment by bsima** [comment]
@VShangxiao It's probably this: https://stackoverflow.com/questions/37227854/pyenv-build-failed-ubuntu-15-04-using-python-build-20160509/37229034#37229034

tl;dr `sudo apt install ca-certificates`

**Comment by Bilida-Mikhail** [comment]
> hmm, forcing a reinstall via
> 
> ```
> sudo apt-get remove libssl-dev
> sudo apt-get install libssl-dev
> ```
> seems to have fixed it.
@tyler274 THANKS!
Helped me
